src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Functions/HealthChecks/Strategies/AzureBlobStorageHealthCheckStrategy.cs

```diff
@@ -0,0 +1,37 @@
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Clients.AzureBlobStorage;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Options;
+using Microsoft.Extensions.Options;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks.Strategies;
+
+internal class AzureBlobStorageHealthCheckStrategy(
+    IAzureBlobStorageClient azureBlobStorageClient,
+    IOptions<AppOptions> appOptions) : IHeathCheckStrategy
+{
+    public async Task<HeathCheckResult> Run(CancellationToken cancellationToken)
+    {
+        var containerName = appOptions.Value.SearchableDocumentsContainerName;
+        if (string.IsNullOrWhiteSpace(containerName))
+        {
+            return new HeathCheckResult(false,
+                $"Azure blob storage container name is not specified in {AppOptions.Section}.{nameof(AppOptions.SearchableDocumentsContainerName)}");
+        }
```

Comment:
Just wondering what you think to removing the validation in the host startup now or do you think we should keep both?

In my experience it will be quite a lot more time consuming to find the right location to view the startup logs and diagnose a startup failure, than to query the healthcheck response.